README.md

```diff
@@ -8,8 +8,9 @@
 
 <p align="center">
   <a href="https://github.com/input-output-hk/cardano-wallet/releases"><img src="https://img.shields.io/github/release-pre/input-output-hk/cardano-wallet.svg?style=for-the-badge" /></a>
-  <a href="https://travis-ci.org/input-output-hk/cardano-wallet"><img src="https://img.shields.io/travis/input-output-hk/cardano-wallet.svg?style=for-the-badge" /></a>
-  <a href="https://buildkite.com/input-output-hk/cardano-wallet-nightly"><img src="https://img.shields.io/buildkite/59ea9363b8526e867005ca8839db47715bc5f661f36e490143.svg?label=NIGHTLY&style=for-the-badge" /></a>
+  <a href="https://buildkite.com/input-output-hk/cardano-wallet"><img src="https://img.shields.io/buildkite/7ea3dac7a16f066d8dfc8f426a9a9f7a2131e899cd96c444cf/master?label=BUILD&style=for-the-badge"/></a>
```

Comment:
Yes, all tests, including integration, are _in_ the `BUILD` badge (https://buildkite.com/input-output-hk/cardano-wallet). The badge points to master.

Reply:
Hmm ok. If it points to https://buildkite.com/input-output-hk/cardano-wallet/builds/1928, 8 minutes seemed so quick though 😄

Reply:
Indeed, `master` seems to skip integration. It seems that integration is included in `bors\staging`. https://buildkite.com/input-output-hk/cardano-wallet/builds/1926 :)

Reply:
I think that pointing the badge to `master` is ok though. If `bors\staging` fails then I suppose that master would fail too. Not scoping to any branch will show the last build’s status, so presumably statuses of our local branches, which is probably not something we'd want.

Reply:
If `bors/staging` fails, then nothing is merged to `master`. In addition, nothing can get into `master` without first going through `bors/staging`. So, when something land on `master`, integration tests have been executed :)

Reply:
I just realised that travis doesn't run _any_ tests anymore though 🙃

Reply:
> So, when something land on master, integration tests have been executed

`master` won't have a chance to fail due to flaky integration tests though. Not sure if that's good or bad.

Reply:
What do you mean? When we do `bors r+`:

- bors checkout a branch `bors/staging` from current `master`
- bors merges one or several branches into `bors/staging`
- bors wait for integration tests to run
- If tests pass, bors replace `master` with `bors/staging`

So in practice, tests have run on top of master and had a chance to fail :upside_down_face: ...

Reply:
Sorry, I meant that the badge doesn't have a chance to show status as failing because of flaky integration tests.

If the tests are flaky, won't we just be unable to merge to master sometimes? And bors might even retry if there are multiple branches being merged?

Reply:
Well, tests should be fixed now with the dynamic port selection. Plus, we don't want the badge to show the result of staging at all time, because staging may fail occasionally and that's totally fine. 
If there's a failure in a test, bors won't retry it so we can still observe "flaky tests" if they occur again.

Reply:
If we'd like to be more granular then maybe go with something like this:

![Screenshot from 2019-09-16 18-05-33](https://user-images.githubusercontent.com/42900201/64974209-a4a8a100-d8ac-11e9-8d32-05e2b02147be.png)